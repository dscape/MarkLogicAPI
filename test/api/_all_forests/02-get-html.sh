#!/bin/sh
curl -sH "Accept: text/html" localhost:6984/_all_forests \
  | grep '<p>Documents</p>' | wc -l
