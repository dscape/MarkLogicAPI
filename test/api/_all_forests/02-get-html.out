<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <title>MarkLogic Server - Forest Listing</title>
  </head>
  <body><p>Triggers</p><p>Last-Login</p><p>Schemas</p><p>Documents</p><p>App-Services</p><p>Security</p><p>Fab</p><p>Modules</p></body>
</html>
